README.md

```diff
@@ -29,3 +29,7 @@ native API header files.
  * [Reporting bugs](docs/bug-reporting.md)
 
 [native-dev]: https://webrtc.googlesource.com/src/+/refs/heads/master/docs/native-code/index.md
+
+### Note
```

Comment:
I am a bit confused whether we should make changes in the original readme file of webrtc, I don't want to modify it, but I still want to explain these contributions from the community through a readme.

Reply:
I think it would be better to create a file like README_webrtc_sdk.txt.

Reply:
I would vote for modifying the main README, first explaining the intent of the repo (i.e. provide mobile SDKs), and the changes we are applying to it.

Then include original Google readme at the bottom.